keyboards/jj40/keymaps/waples/keymap.c

```diff
@@ -6,15 +6,13 @@ extern keymap_config_t keymap_config;
 
 #define _QWERTY 0
```

Comment:
You should turn these layer defines into an enum so you don't have to keep renumbering them :)

Reply:
How do I do that? :P

Reply:
You'd use:
```c
enum layers {
    _QWERTY,
    _DVORAK,
    _LEFTY,
    _RIGHTY,
    _DUAL,
};
```

It defaults to 0, for the first layer, and automatically increments.

Reply:
```c
enum layers {
    _QWERTY,
    _DVORAK,
    _LEFTY,
    _RIGHTY,
    _DUAL,
}:
```

This is programmatically identical to the `#define`s you have now.